ajax/libs/simplemodal/package.json

```diff
@@ -3,5 +3,22 @@
   "filename": "jquery.simplemodal.min.js",
   "version": "1.4.4",
   "description": "SimpleModal is a lightweight jQuery Plugin which provides a powerful interface for modal dialog development.",
-  "homepage": "http://www.ericmmartin.com/projects/simplemodal/"
+  "homepage": "http://www.ericmmartin.com/projects/simplemodal/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ericmmartin/simplemodal"
+  },
+  "keywords": [
+    "eric martin",
+    "simplemodal",
+    "wordpress",
+    "jquery",
+    "ui architect",
+    "web standards",
+    "usability",
+    "consulting",
+    "php",
+    "modal"
+  ],
+  "license": "MIT"
```

Comment:
Thought there is MIT license in repo, I guess they are using GPL?

See https://github.com/ericmmartin/simplemodal/blob/master/bower.json